thinkshout/bene-project
# 4 fix install

to test:

view updated README. Install should be able to be done via

`./scripts/install.sh -d /some/bene/project`

NOTES:
the post-install hook of `drush mi menu_links`–specified in RoboFile.php–will always fail (drush command not found). This has never worked for me. I would love to hear theories on that, but it doesn't affect overall install.
